drivers/kw41zrf/include/kw41zrf_intern.h

```diff
@@ -27,18 +27,10 @@
 /* For ZLL CPU registers */
 #include "cpu.h"
 
-#ifdef MODULE_PM_LAYERED
-#include "pm_layered.h"
-#define PM_BLOCK(x) pm_block(x)
-#define PM_UNBLOCK(x) pm_unblock(x)
 /* When the transceiver is not in DSM, this power mode will be blocked.
  * TODO: Change this to symbolic name KINETIS_PM_VLPS when Kinetis power
- * management is merged (https://github.com/RIOT-OS/RIOT/pull/7897) */
+ * management is merged (https://github.com/RIOT-OS/RIOT/pull/13742) */
```

Comment:
Hmm just saw this, wrong PR number?

Reply:
But #13742 is this PR and it introduces `KINETIS_PM_VLPS`

Reply:
Whoops, I'm not sure why I updated the PR number instead of executing the TODO. I'll fix that.